### DISCUSSION TITLE
GrandOrgue on Manjaro

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/grandorgue#952] [Help]
What is the best option to install GrandOrgue on Manjaro? Until now I have used the AppImage. But that does not integrate well with the program launcher.

**Comment by larspalo** [comment]
Do you have the appimagelauncher installed?

From what I just very quickly have read, it seems to be Arch Linux based. If you're somewhat technical it's fairly easy to build GO yourself from the sources. Instructions in https://github.com/GrandOrgue/grandorgue/blob/master/BUILD.md if you want to have a go at it.

**Comment by lbssousa** [comment]
GrandOrgue used to have a PKGBUILD for Arch LInux. Any chance to bring it back? You may also ask current maintainer of package grandorgue-svn on AUR to keep it up to date.

**Comment by oleg68** [comment]
@lbssousa We make linux packages with cmake/cpack. So we'll return pkgbuild when iw is supported by cmake. See https://gitlab.kitware.com/cmake/cmake/-/merge_requests/4185 for more details.
